### ISSUE TITLE
Loading a config file is slow 

### ISSUE THREAD

**Issue by ArnaudBarre** [windicss/vite-plugin-windicss#284]
When adding this plugin, vite startup time goes from ~400ms to ~550ms without config file to  ~700ms with config file.

I don't know what causes the first slow down, but for the second one it seems to be how the config is compiled. And caching doesn't seems to help a lot on restart. I think this plugin could use esbuild to bundle the config instead of jiti. I can try a PR if you want.
 
<img width="540" alt="Screenshot 2022-02-23 at 20 23 04" src="https://user-images.githubusercontent.com/14235743/155392284-f3346c1b-314d-44e0-98a5-cd057c767025.png">

**Comment by jinzhubaofu** [comment]
@ArnaudBarre I got this problem too. I figured out a solution which maybe can help you.

TLDR: you should keep `windi.config.ts` as easy as possible. Put your configurations which depends on 3rd party npm package in your `vite.config.ts`.

```
import {defineConfig} from 'vite';
export default defineConfig({
  // ...
  plugins: [
    windicss({
      async onConfigResolved(config) {}
    })
  ]
});
```

Here is why:
1. vite-plugin-windicss will load your `windi.config.ts` using `jiti`.
2. `jiti` will transform all the depedencies of `windi.config.ts` with `babel`, to compile ts into js and make sure it's cjs format.

So, `jiti` will cost a lot time.
